src/BlogDetails.js

```diff
@@ -0,0 +1,38 @@
+import { useParams } from "react-router";
+import useFetch from './useFetch';
+import { useHistory } from 'react-router-dom';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/7bel6/b-blog/pullRequest?prid=7919486)
```suggestion
import { useHistory } from "react-router-dom";
```